e2e-tests/development-runtime/gatsby-config.js

```diff
@@ -7,7 +7,7 @@ module.exports = {
     },
   },
   flags: {
-    DEV_SSR: false,
+    DEV_SSR: true,
```

Comment:
We shouldn't do this here (not yet at least) - https://github.com/gatsbyjs/gatsby/tree/master/integration-tests/ssr are the test specifically for DEV_SSR